### PR TITLE
Annotate NotificationResult

### LINKED ISSUE #26: Anotate NotificationResult properly
To improve typing annotation, we should properly annotate `NotificationResult` by using [NamedTuple](https://mypy.readthedocs.io/en/stable/kinds_of_types.html#named-tuples).

### PR DESCRIPTION
**Pull request by tpazderka**
Close #26

### CODE REVIEW

File: cz_nia/message.py [hunk 1]
```diff
@@ -185,7 +184,10 @@ def extract_message(self, response):
         return None
 
 
-NotificationResult = namedtuple('NotificationResult', 'notifications last_id more_notifications')
+NotificationResult = NamedTuple('NotificationResult',
+                                [('notifications', List),
```
Comment (fenryxo) [inline cz_nia/message.py:188]: Do we know the type of the list elements?

Comment (tpazderka) [reply cz_nia/message.py:188]: Probably `Dict[str, Union[datetime.datetime, str]]`

Comment (fenryxo) [reply cz_nia/message.py:188]: I'd then change `List` to `List[Dict[str, Any]]` or with the union.

Comment (tpazderka) [reply cz_nia/message.py]: Aded

**Pull request merged**